raft/src/main/java/org/apache/kafka/raft/NetworkChannel.java

```diff
@@ -30,10 +30,8 @@
     int newCorrelationId();
 
     /**
-     * Send an outbound message. This could be either an outbound request
+     * Send an outbound message. This could be an outbound request
      * (i.e. an instance of {@link org.apache.kafka.raft.RaftRequest.Outbound})
-     * or a response to a request that was received through {@link #receive(long)}
```

Comment:
The receive() has been removed in #9732.

Reply:
How about just:

```java
/**
 * Send an outbound request message.
 *
 * @param request outbound request to send
 */
```